sycl/test-e2e/PrivateAlloca/private_alloca_bool_size.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %{build} -w -o %t.out
+// RUN: echo 1  | %{run} %t.out
```

Comment:
Should we try different sizes?

Reply:
All non-zero values would be equivalent to 1 as we're using `bool` as a type. `0` is not allowed by the extension, so we're basically covering all values here.